SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -3,48 +3,55 @@ import Foundation
 extension String {
 
     public subscript(range: Range<Int>) -> String {
-        let chars = Array(characters)
-        let substringCharacters = chars[range]
-        return String(substringCharacters)
+        let lowerBound = index(startIndex, offsetBy: range.lowerBound)
```

Comment:
In Swift 4, can't this entire method simply be implemented as
`return String(self[range])`? And similarly for some of the other methods?

Reply:
unfortunately no. you can use subscript range between String.index ranges. The other methods in this class actually use this class :)